src/internal/testpachd/mock_transaction.go

```diff
@@ -145,7 +147,7 @@ func (api *ppsTransactionAPI) UpdateJobStateInTransaction(txnCtx *txncontext.Tra
 	return errors.Errorf("unhandled pachd mock: pps.UpdateJobStateInTransaction")
 }
 
-func (api *ppsTransactionAPI) CreatePipelineInTransaction(txnCtx *txncontext.TransactionContext, req *pps.CreatePipelineRequest) error {
+func (api *ppsTransactionAPI) CreatePipelineInTransaction(ctx context.Context, txnCtx *txncontext.TransactionContext, req *pps.CreatePipelineRequest) error {
```

Comment:
Note that despite the similar names, `context.Context` and `txncontext.TransactionContext` are very different things; `txncontext.TransactionContext` does _not_ implement `context.Context`.